src/posthog-core.ts

```diff
@@ -169,14 +169,14 @@ const defaultConfig = (): PostHogConfig => ({
 })
 
 /**
- * create_mplib(token:string, config:object, name:string)
+ * create_phlib(token:string, config:object, name:string)
```

Comment:
👨‍🍳 👌

Reply:
I applaud the change but it seems quite irrelevant to this PR per se, and has caused some unnecessary merge conflicts. Small independent `refactor:` PRs for the win, in the future ;)